skbio/metadata/__init__.py

```diff
@@ -16,4 +16,12 @@
 # ----------------------------------------------------------------------------
 
 from skbio.util import TestRunner
+
+from ._interval import IntervalMetadata
+from ._mixin import (PositionalMetadataMixin, MetadataMixin)
+
+__all__ = ['IntervalMetadata',
+           'MetadataMixin', 'PositionalMetadataMixin',
```

Comment:
Remove MetadataMixin, PositionalMetadataMixin, and IntervalMetadataMixin from `__all__` (the latter doesn't exist, and all mixins are private classes)